socs/agents/pysmurf_controller/agent.py

```diff
@@ -767,7 +849,37 @@ def bias_dets(self, session, params):
 
             session.data['biases'] = biases.tolist()
 
-            return True, "Finished taking bias steps"
+            return True, "Finished biasing detectors"
+
+    @ocs_agent.param('disable_amps', default=True, type=bool)
+    @ocs_agent.param('disable_tones', default=True, type=bool)
+    def all_off(self, session, params):
+        """all_off(disable_amps=True, disable_tones=True)
+
+        **Task** - Turns off tones, flux-ramp voltage and amplifier biases
+
+        Args
+        -------
+        disable_amps: bool
+            If True, will disable amplifier biases
+        disable_tones: bool
+            If True, will turn off RF tones and flux-ramp signal
+        """
+        if params['kwargs'] is None:
+            params['kwargs'] = {}
```

Comment:
Is this needed? Doesn't look like `params['kwargs']` is passed to anything later.

Reply:
Ah yeah this is probably a mistake